src/main/java/Chacha.java

```diff
@@ -6,15 +6,43 @@ public static void main(String[] args) {
         System.out.println("Hello! I'm Chacha\n" + "What can I do for you?");
         Scanner input = new Scanner(System.in);
         String s = input.nextLine();
-        ArrayList<String> list = new ArrayList<String>();
+        ArrayList<Task> taskList = new ArrayList<Task>();
         while (!s.equals("bye")) {
             if (s.equals("list")) {
-                for (int i = 0; i < list.size();i++) { 		      
-	                System.out.println(i + 1 + ". " + list.get(i)); 		
+                for (int i = 0; i < taskList.size();i++) {
+                    Task t = taskList.get(i);	      
+	                System.out.println(i + 1 + 
+                        ".[" + 
+                        t.getStatusIcon() + 
+                        "] " + 
+                        t.getDescription()); 		
 	            }   
+                 
+            } else if (s.contains("unmark")) {
```

Comment:
Can consider using s.startsWith function to check if the task type is indeed the first input